### PR TITLE
How to protect yourself from scams

### LINKED ISSUE #991: Add "How to Protect Yourself from Scams" page

### PR DESCRIPTION
**Pull request by krichard410**
Closes #991 

Open for more tips/advice on this subject =)

### CODE REVIEW

File: docs/learn-scams.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+### Always check the source
+
+For any potential scam, always be sure to do a background check on the source, i.e, look at any
+username, email, Youtube channel name, etc. If something seems fishy, that's because it likely is.
+
+### Check twice before sending DOT/KSM
+
+A good practice to take in consideration is to verify the address you are sending crypto to. If you
+don't know that account, you probably shouldn't be sending your assets to there. It is very
+important to understand that crypto is a decentralized space, if you lose your assets, that is on
```
Comment (krichard410) [inline docs/learn-scams.md]: Does it sound too harsh here if I bluntly say 'that is on you' ?

Comment (krichard410) [reply docs/learn-scams.md]: perhaps it'd sound better to say 'it is your loss' ?

File: docs/learn-scams.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+### Types of Scams
+
+- Personal messages over Telegram, and other social medias
+- 'Send us DOT/KSM, we'll send you double'
+- Emails asking for DOT/KSM addresses, posing as a member of our team
+
+### Admin will never contact you over Telegram
+
+If you've received a message from an admin over Telegram, ignore it. Our Polkadot team will never
+personally message you. Our social media's are posted on our website and any new social media's will
```
Comment (laboon) [inline docs/learn-scams.md]:
```suggestion
personally message you first. Our social media's are posted on our website and any new social media's will
```

Comment (laboon) [inline docs/learn-scams.md]: Not sure what "our social media's" are.  Our social media accounts?

Link to the website (polkadot.network)

Add: we will never offer to sell you DOTs at a discount, air-drop "rewards", or message you privately to help with a problem you posted publicly.

File: docs/learn-scams.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+be announced by our team.
+
+### Always check the source
+
+For any potential scam, always be sure to do a background check on the source, i.e, look at any
+username, email, Youtube channel name, etc. If something seems fishy, that's because it likely is.
```
Comment (laboon) [inline docs/learn-scams.md]: add "Feel free to check with support@polkadot.network" again

File: docs/learn-scams.md [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+
+### Check twice before sending DOT/KSM
+
+A good practice to take in consideration is to verify the address you are sending crypto to. If you
+don't know that account, you probably shouldn't be sending your assets there. It is very important
+to understand that crypto is a decentralized space, if you lose your assets, that is on you.
+
+### Keep your personal data secure
+
+One should _never_, share their secret key with anyone, nor any other personal data that is
```
Comment (laboon) [inline docs/learn-scams.md]:
```suggestion
One should _never_, share their secret key, seed, mnemonic phrase, etc. with anyone, nor any other personal data that is
```

File: docs/learn-scams.md [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+associated with you. It's recommended to store your information safe on paper or on a hardware
+wallet.
```
Comment (laboon) [inline docs/learn-scams.md]: Never enter your seed or mnemonic phrase on a website.

File: docs/learn-scams.md [hunk 6]
```diff
@@ -0,0 +1,38 @@
...
+yourself and your non-refundable crypto assets from scammers. If you ever feel you are being
+scammed, please email [our support team](mailto:support@web3.foundation).
+
+### Types of Scams
+
+- Personal messages over Telegram, and other social medias
+- 'Send us DOT/KSM, we'll send you double'
+- Emails asking for DOT/KSM addresses, posing as a member of our team
+
+### Admin will never contact you over Telegram
```
Comment (laboon) [inline docs/learn-scams.md]:
```suggestion
### Admins will never contact you over Telegram, Twitter, etc.
```

File: docs/learn-scams.md [hunk 7]
```diff
@@ -0,0 +1,38 @@
...
+
+Scams have unfortunately made an impact on our community. It's important to stay alert and protect
+yourself and your non-refundable crypto assets from scammers. If you ever feel you are being
+scammed, please email [our support team](mailto:support@web3.foundation).
+
+### Types of Scams
+
+- Personal messages over Telegram, and other social medias
+- 'Send us DOT/KSM, we'll send you double'
+- Emails asking for DOT/KSM addresses, posing as a member of our team
```
Comment (laboon) [inline docs/learn-scams.md]: They won't ask for addresses, they'll ask for private keys/seeds/etc.  Addresses by themselves don't do much.  Or they will point users to sites that are copies of real sites but that ask to enter seeds.

File: docs/learn-scams.md [hunk 8]
```diff
@@ -0,0 +1,38 @@
...
+
```
Comment (laboon) [inline docs/learn-scams.md:23]: Add a bit about how scammers will take official video and add text so that it still looks like an official video.

Think about it - is it too good to be true?  Then it probably is.

File: docs/learn-scams.md (hunk 3 above)
Comment (laboon) [inline docs/learn-scams.md]: Many scammers will create nearly perfect imitations of sites - triple-check the URL!  Never enter any personal data after clicking on a link, always type it in yourself.

File: docs/learn-scams.md [hunk 9]
```diff
@@ -0,0 +1,38 @@
...
+### Always check the source
+
+For any potential scam, always be sure to do a background check on the source, i.e, look at any
+username, email, Youtube channel name, etc. If something seems fishy, that's because it likely is.
+
+### Check twice before sending DOT/KSM
+
+A good practice to take in consideration is to verify the address you are sending crypto to. If you
+don't know that account, you probably shouldn't be sending your assets there. It is very important
+to understand that crypto is a decentralized space, if you lose your assets, that is on you.
```
Comment (laboon) [inline docs/learn-scams.md]:
```suggestion
to understand that crypto is a decentralized space.  If you transfer your assets to a different address, there is nobody who can get your funds back besides the account to which you sent it..
```

File: docs/learn-scams.md (hunk 7 above)
Comment (laboon) [inline docs/learn-scams.md]: Add: 
People offering to help you stake or get rewards
People answering questions you asked publicly in a private chat

File: docs/learn-scams.md [hunk 10]
```diff
@@ -0,0 +1,38 @@
...
+
+### Types of Scams
+
+- Personal messages over Telegram, and other social medias
+- 'Send us DOT/KSM, we'll send you double'
+- Emails asking for DOT/KSM addresses, posing as a member of our team
+
+### Admin will never contact you over Telegram
+
+If you've received a message from an admin over Telegram, ignore it. Our Polkadot team will never
```
Comment (laboon) [inline docs/learn-scams.md:49]:
```suggestion
If you've received a message from a "Polkadot admin" or "Polkadot support", block and report it. Our Polkadot team will never
```

Comment (laboon) [reply docs/learn-scams.md:49]: Telegram is a huge source of these scams but not the only one

File: docs/learn-scams.md [hunk 11]
```diff
@@ -0,0 +1,38 @@
+---
+id: learn-scams
+title: How to Protect Yourself from Scams
+sidebar_label: How to Protect Yourself from Scams
+---
+
+Scams have unfortunately made an impact on our community. It's important to stay alert and protect
+yourself and your non-refundable crypto assets from scammers. If you ever feel you are being
+scammed, please email [our support team](mailto:support@web3.foundation).
+
```
Comment (laboon) [inline docs/learn-scams.md:10]: Essential rules:
1. Never, ever, ever share your seed phrase or account password
1. Do not trust anyone online.  It is trivial for them to lie, change identities.
1. If you are scammed, there is likely nothing that can be done to recover your funds.  If a scammer gets a hold of your seed phrase, they can transfer all of your funds to their account in seconds.  It is better to be safe than to risk all of DOT and KSM.
1. If it sounds too good to be true, it probably is.  People, especially celebrities, do not give away crypto for free.  Even if they wanted to, they could just ask for your address as opposed to having you send them tokens.

File: docs/learn-scams.md [hunk 12]
```diff
@@ -0,0 +1,38 @@
...
+### Types of Scams
+
+- Personal messages over Telegram, and other social medias
```
Comment (laboon) [inline docs/learn-scams.md]:
```suggestion
- Personal messages over Telegram, Twitter, and other social media
```

"media" is already the plural of "medium" =)

File: docs/learn-scams.md (hunk 11 above)
Comment (laboon) [reply docs/learn-scams.md:10]: 5. Scams are absolutely rife in this space.  It is easy and cheap to set a scam up, and hard to shut one down.  Therefore, the onus is on the user to be as diligent as possible in avoiding them.

**Review by laboon — CHANGES_REQUESTED** [review]

File: docs/learn-scams.md (hunk 7 above)
Comment (laboon) [reply docs/learn-scams.md]: Add:

Scammers will often make it seem like the 'deal' is only available for a limited time.  Do not be tricked by this, it is always better to confirm than to risk losing everything.

File: docs/learn-scams.md (hunk 1 above)
Comment (lsaether) [reply docs/learn-scams.md]: I would rephrase to say something like "It's your responsibility to make sure that you understand where you are sending your funds to, crypto is a decentralized space and your only recourse if a mistake is made is by appealing to the council (who usually will not get involved in matters of mistaken transfers)."

File: docs/learn-scams.md [hunk 13]
```diff
@@ -0,0 +1,62 @@
...
+If you've received a message from an admin over Telegram, ignore it. Our Polkadot team will never
+personally message you. Our social media's are posted on our website and any new social media's will
+be announced by our team. We will never offer to sell you DOTs at a discount, air-drop "rewards", or
+message you privately to help with a problem you posted publicly.
+
+### Always check the source
+
+For any potential scam, always be sure to do a background check on the source, i.e, look at any
+username, email, Youtube channel name, URL, etc. If something seems fishy, that's because it likely
+is. Never enter any personal data after clicking on a link, always type it in yourself. Feel free to
```
Comment (Swader) [inline docs/learn-scams.md]: Not clear what "always type it in yourself" refers to, looks like Personal data. Rephrase or separate sentences.

File: docs/learn-scams.md [hunk 14]
```diff
@@ -0,0 +1,62 @@
...
+check with support@polkadot.network.
```
Comment (Swader) [inline docs/learn-scams.md]: Also add mailto link

**Review by Swader — CHANGES_REQUESTED** [review]

File: docs/learn-scams.md [hunk 15]
```diff
@@ -10,29 +10,53 @@ scammed, please email [our support team](mailto:support@web3.foundation).
...
-be announced by our team.
+be announced by our team. We will never offer to sell you DOTs at a discount, air-drop "rewards", or
+message you privately to help with a problem you posted publicly.
 
 ### Always check the source
 
 For any potential scam, always be sure to do a background check on the source, i.e, look at any
-username, email, Youtube channel name, etc. If something seems fishy, that's because it likely is.
+username, email, Youtube channel name, URL, etc. If something seems fishy, that's because it likely
+is. Never enter any personal data after clicking on a link, always type personal data in yourself.
```
Comment (laboon) [inline docs/learn-scams.md]: "Never enter any personal data after clicking on a link, always type personal data in yourself."

What do you mean here?

File: docs/learn-scams.md [hunk 16]
```diff
@@ -8,24 +8,54 @@ Scams have unfortunately made an impact on our community. It's important to stay
...
+tricked by this, it is always better to confirm than to risk losing everything.
 
 ### Admin will never contact you over Telegram, nor any other Social Media
 
 If you've received a message from an admin over Telegram, ignore it. Our Polkadot team will never
-personally message you. Our social media's are posted on our website and any new social media's will
-be announced by our team. We will never offer to sell you DOTs at a discount, air-drop "rewards", or
-message you privately to help with a problem you posted publicly.
+personally message you. Our social media accounts are posted on our website and any new social media
+accounts will be announced by our team. We will never offer to sell you DOTs at a discount, air-drop
```
Comment (krichard410) [inline docs/learn-scams.md]:
```suggestion
accounts will be announced by our team. We will never offer to sell you DOT at a discount, air-drop
```

File: docs/learn-scams.md [hunk 17]
```diff
@@ -8,24 +8,54 @@ Scams have unfortunately made an impact on our community. It's important to stay
...
+4. If it sounds too good to be true, it probably is. People, especially celebrities, do not give
+   away crypto for free. Even if they wanted to, they could just ask for your address as opposed to
+   having you send them tokens.
+5. Scams are absolutely rife in this space. It is easy and cheap to set a scam up, and hard to shut
+   one down. Therefore, the onus is on the user to be as diligent as possible in avoiding them.
+
+### Some Common Types of Scams
+
+- Private messages sent to you over Telegram, Twitter, and other social media
+- "Giveaways" advertising that you 'send us some DOT/KSM, we'll send you double back'
```
Comment (krichard410) [inline docs/learn-scams.md]:
```suggestion
- "Giveaways" advertising that you "send us some DOT/KSM, we'll send you double back"
```
nitpick
